spec/factories/gardens.rb

```diff
@@ -1,5 +1,8 @@
 FactoryBot.define do
   factory :garden do
-    
+    latitude { Faker::Address.latitude }
+    longitude { Faker::Address.longitude }
+    name { Faker::Address.country }
+    description { "This is a description" }
   end
```

Comment:
Great use of Faker. Be weary of the latitudes and longitudes. I used them with Faker as well and had a conflict with Travis CI because on one of my tests had too many digits in the longitude and Travis would not allow for a number that long. If you start failing with Travis you may need to manually input a latitude or longitude.

Reply:
Thanks for the heads up